_posts/2014-11-13-cockpit-plugin-tutorial.md

```diff
@@ -33,25 +35,22 @@ Components, and more specifically their HTML and Javascript files, live in packa
             "label": "Pinger",
             "path": "ping.html"
         }
-    },
-    "content-security-policy": "default-src 'self' 'unsafe-inline' 'unsafe-eval'"
+    }
 }
 ```
 
-The manifest above has a `"tools"` subsection. Each tool is listed in the *Tools* menu by Cockpit. The `"path"` is the name of the HTML file that implements the tool, and the `"label"` is the text to show in the *Tools* menu.
+The manifest above has a `"tools"` subsection, which means that it will appear in the lower section of the menu (see the [manifest documentation](https://cockpit-project.org/guide/latest/packages.html#package-manifest) for details). Each tool is listed in the menu by Cockpit. The `"path"` is the name of the HTML file that implements the tool, and the `"label"` is the text to show in the menu.
 
-You'll notice that we haven't told Cockpit about the package yet. To do so you either place or symlink the package into one of two places:
+You'll notice that we haven't told Cockpit about how to find the package yet. To do so you either place or symlink the package into one of two places:
```

Comment:
```suggestion
You'll notice that we haven't told Cockpit about how to find the package yet. To do so you either move or symlink the package into one of two places:
```

Reply:
We probably want something more specific than "place" here. I'm not sure if "move" is right... but perhaps it is?

Reply:
"copy", I think. I'll add you suggestion and squash in the s/move/copy/.